YourToDo/urls.py

```diff
@@ -35,15 +38,11 @@
     url(r'^about/$',    'YourToDo.views.about'),
     url(r'^contact/$',    'YourToDo.views.contact'),
     url(r'^contact_success/$', 'YourToDo.views.contact_success'),
-
-    # User Authentication Urls
-    url(r'^login/$',    'YourToDo.views.login'),
-    url(r'^logout/$',   lambda request: logout_then_login(request, "/"), name='logout'),
-
-    # User Registration Urls
-    url(r'^registration/$', 'YourToDo.views.registration'),
-    url(r'^registration_success/$', 'YourToDo.views.registration_success'),
     
+    # Django Registration Redux Urls
+    url(r'^accounts/logout/', 'YourToDo.views.ytdLogout', name='auth_logout'),
```

Comment:
Any reason we called this ytdLogout? Where I come from, we actually use reasonable names. Please fix this immediately.